docs/cis_self_assessment.md

```diff
@@ -735,9 +741,20 @@ Ensure that the admission control plugin SecurityContextDeny is set if PodSecuri
 SecurityContextDeny can be used to provide a layer of security for clusters which do not have PodSecurityPolicies enabled.
 </details>
 
-**Result:** Pass
+**Result:** **Not Scored - Operator Dependent**
+
+**Audit:**
+Run the below command on the master node.
+
+```bash
+/bin/ps -ef | grep kube-apiserver | grep -v grep
+```
+
+Verify that the `--enable-admission-plugins` argument is set to a value that includes `SecurityContextDeny`, if `PodSecurityPolicy` is not included.
 
 **Remediation:**
+By default, RKE2 only sets `NodeRestriction,PodSecurityPolicy` as the parameter to the `--enable-admission-plugins` argument.
```

Comment:
The remediation for this one should be a little more nuanced and explain that we enable the PodSecurityPolicy by default.
```suggestion
By default, RKE2 automatically enables the PodSecurityPolicy admission plugin. Therefore, the `SecurityContextDeny` plugin need not be enabled.
```